README.md

```diff
@@ -77,4 +77,20 @@ How to run the container
 ```
 docker run --publish 5000:5000 --env-file .env todo-app:prod
 docker run --env-file ./.env -p 5100:5000 --mount "type=bind,source=$(pwd)/todo_app,target=/app/todo_app" todo-app:dev
-```
\ No newline at end of file
+```
+
+Build the test container image
+ 
+```
+docker build --target test --tag todo-app:test
```

Comment:
I think you're missing the final argument for the build context here - that pesky `.`!
```suggestion
docker build --target test --tag todo-app:test .
```